src/main/java/com/google/javascript/cl2dts/DeclarationGenerator.java

```diff
@@ -130,8 +203,11 @@ public void println(CheckLevel level, JSError error) {}
       protected void printSummary() {}
     });
 
+    if (externs.isEmpty()) {
+      externs = opts.skipParseExterns ? Collections.<SourceFile>emptyList() : getDefaultExterns();
```

Comment:
throw if there are externs explicitly passed and skipParseExterns is true (or set at all)?